src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,36 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public boolean isDateInRange(String date, LocalDate startDate, LocalDate endDate) {
```

Comment:
if this method designed to be used only inside of this class let's make it private